tools/depends/target/gnutls/Makefile

```diff
@@ -1,23 +1,16 @@
 include ../../Makefile.include
-DEPS= ../../Makefile.include Makefile size-max.patch add-dl-as-private-lib.patch
+DEPS= ../../Makefile.include Makefile size-max.patch add-dl-as-private-lib.patch disable-hw-accel-aarch64.patch
 
 # lib name, version
 LIBNAME=gnutls
 VERSION=3.6.11.1
 SOURCE=$(LIBNAME)-$(VERSION)
 ARCHIVE=$(SOURCE).tar.xz
 
-ifeq (darwin, $(findstring darwin, $(HOST)))
-# darwins tar doesn't know about xz - so we need our native version of tar here
-ARCHIVE_TOOL=$(ARCHIVE_TOOL_NATIVE)
```

Comment:
How is this change related to fixing the host triplet to `aarch64-apple-darwin`?

Reply:
Moving to separate commit

Reply:
Isn't this variable (`ARCHIVE_TOOL_NATIVE`) now unused?

Reply:
So remove tar altogether from native depends?

Reply:
Or is it needed to other stuff?

Reply:
If it is unused or not required, yes.

Reply:
tar needed for dpkg on ios/tvos so it stays I guess.

https://github.com/xbmc/xbmc/blob/1eff5afda088f8fc58b63f8a7313c2b71016b3e2/tools/depends/native/Makefile#L16-L18

Reply:
Is it really needed by dpkg, or did only previously system tar versions miss some features?

Reply:
Let’s remove it and see

Reply:
Fails on darwin_embedded with it removed. Must be required.

Reply:
Hmm, or not. I forgot to remove the dependency.

Reply:
tar/xz still needed for ios/tvos.